examples/migrationExample/src/main/java/io/realm/examples/realmmigrationexample/MigrationExampleActivity.java

```diff
@@ -59,7 +60,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // When you create a RealmConfiguration you can specify the version of the schema.
         // If the schema does not have that version a RealmMigrationNeededException will be thrown.
-        RealmConfiguration config0 = new RealmConfiguration.Builder(this)
+        RealmConfiguration config0 = new RealmConfiguration.Builder()
                 .name("default0")
```

Comment:
Actually it is not a requirement they end with `.realm`, but it is a nice convention.